index.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+(function() {
+
+  var _generateOutput = require("./generator.js");
+
+  var _stringsArray = [];
+  var _outputArray;
+
+  var FizzBuzz = function(string1, string2) {
+    _stringsArray.push(string1, string2);
```

Comment:
Rather than simply pushing your values in to your `_stringsArray`, when your constructor is run you might want to also reset your `_stringsArray`; the current implementation actually keeps it from allowing different instances to have different output. Currently, this is what's happening:

```javascript
var FizzBuzz = require("sea-d44-fizz-buzz-hl");
var buzz1 = new FizzBuzz();
buzz1.input(1,10);
console.log(buzz1.output()); // >> [1,2,"fizz",4,"buzz",...]

var buzz2 = new FizzBuzz("tick", "tock");
buzz2.input(1,100);
console.log(buzz2.output()); // >> [1,2,"fizz",4,"buzz"...]
```

Basically, with these private variables, you're sharing your data between the instances (which can be useful in some cases). It might make more sense to store the `_stringsArray` as a property of the instance: `this.stringsArray` instead:

```javascript
var FizzBuzz = function(string1, string2) {
  this.stringsArray = [string1, string2];
};
```